docs/api/tags.md

```diff
@@ -12,6 +12,7 @@ state`items.${props`itemKey`}`
 This allows you to express signals and component dependencies more effectively. There are 4 tags in Cerebral.
 
 - **state** - Used in signals and connect to target state
+- **module** - Used in signals and connect to target state of the module running the signal
```

Comment:
How do you use use 'module' in connect as the view is not bound to a specific module.

Reply:
@henri-hulski It is only for use in operators. should note that :)